Sources/Body.swift

```diff
@@ -90,14 +90,14 @@ extension Body {
         switch self {
         case .buffer(let data):
             let closure: (Stream throws -> Void) = { sender in
-                try sender.send(data)
+                try sender.send(data, timingOut: -1)
```

Comment:
You can actually use `try sender.send(data, timingOut: .never)` instead. I think it’s more clear.